src/ssa/local_ssa.cpp

```diff
@@ -1644,3 +1647,13 @@ bool local_SSAt::can_reuse_symderef(
 
   return true;
 }
+
+void local_SSAt::disable_unsupported_instructions(locationt loc)
+{
+  if (loc->is_other())
+  {
+    auto st = loc->get_code().get_statement();
+    if(st=="array_copy" || st=="array_replace")
```

Comment:
Perhaps we could file an issue for support of memcpy?

Reply:
Done: #160.